proposals/Exceptions.md

```diff
@@ -1,302 +1,17 @@
 # Exception handling
 
 So far there have been two proposals ([first proposal](old/Exceptions.md),
-[second proposal](old/Level-1.md)) proposed. We are going to use the second
-proposal for the binary section spec, but we have not decided on which one to
-use for the instruction part of the spec. This document describes status quo of
-the exception handling proposal: for some parts we present both candidate
-options until we fully decide.
+[second proposal](old/Level-1.md)) proposed. In the TPAC meeting in Oct 2018, we
```

Comment:
I might remove the "so far", and just say something along the lines of "there were two alternatives [links] for the design of EH and we decided on first-class exceptions etc", since that wording is a bit more definitive, reflecting the fact that we've actually made the decision. Then you can probably remove the last sentence in this paragraph since it will be sort of redundant.

Reply:
Done